src/services/blocks/BlocksService.ts

```diff
@@ -437,8 +437,15 @@ export class BlocksService extends AbstractService {
 				) {
 					// multiSig.asMulti.args.call is an OpaqueCall (Vec<u8>) that we
 					// serialize to a polkadot-js Call and parse so it is not a hex blob.
-					const call = this.api.createType('Call', argument.toHex());
-					newArgs[paramName] = this.parseGenericCall(call);
+					try {
+						const call = this.api.createType(
+							'Call',
+							argument.toHex()
+						);
+						newArgs[paramName] = this.parseGenericCall(call);
```

Comment:
How deep can this go? It would be good to have a recursion limit and give up after `n` levels of recursion or someone could submit crazy payloads and take down sidecar when the stack blows?

Reply:
This goes to an arbitrary depth. However it only processes extrinsics in blocks that the node may have. So presumably Substrate would never have an extrinsic with an insane depth of nested call arguments that would blow a node.js stack.